### ISSUE TITLE
Feature request: follow-up messages

### ISSUE THREAD

**Issue by sasa** [rykov/paperboy#6]
Hi,

amazing project. Really looking forward to what paperboy will be capable of in the future.

My feature request would be to send follow-up messages, also known as drip campaigns.

Is this something that will make it as a feature in paperboy some day?

-Sasa

**Comment by rykov** [comment]
@sasa Yes, I'd be interested in having that in the future, however, there are quite a few things on the broadcast/newsletter side that need to be done before shifting focus to other email marketing features.

**Comment by sasa** [comment]
@rykov That sounds great! Looking forward to this, even if it is going to take a little while.
